### DISCUSSION TITLE
Failed to load ImageMagick (MagickWand)" Even though it's on my system

### DISCUSSION THREAD

**Discussion by miloslavnosek** [3rd/image.nvim#192] [Q&A]
Hello, I keep getting this error and I have no idea how to fix it and would much appreciate any kind of help. I'm running openSUSE Tumbleweed and LazyVim distribution. This is the error:

```
image.nvim: magick rock not found, please install it and restart your editor. Error: ".../lazy-rocks/image.nvim/share/lua/5.1/magick/wand/lib.lua:220: Failed to load ImageMagick (MagickWand)"
```

but my system has it installed

```
$ ls ~/.luarocks/share/lua/5.1/magick/wand/
data.lua  image.lua  lib.lua

$ luarocks show magick

magick 1.6.0-1 - Lua bindings to ImageMagick & GraphicsMagick for LuaJIT using FFI

$ magick --version
Version: ImageMagick 7.1.1-34 Q16-HDRI x86_64 22301 https://imagemagick.org
```

my config:
```

return {
  --- Image preview in neovim
  {
    "3rd/image.nvim",
    config = function()
      local img = require("image")
      img.setup({
        backend = "ueberzug",
        integrations = {
          markdown = {
            enabled = true,
            clear_in_insert_mode = false,
            download_remote_images = true,
            only_render_image_at_cursor = true,
            filetypes = { "markdown", "vimwiki", "org" }, -- markdown extensions (ie. quarto) can go here
          },
        },
      })
    end,
  },
}
```

**Comment by 3rd** [comment]
Hey, this seems like an issue with the magick rock, there are some discussions here: https://github.com/3rd/image.nvim/issues/18
I only encountered it once and the solution I found back then was a manual patch.
Sorry about this, working on including better image processing solutions to get rid of the rock.

**Reply by miloslavnosek** [reply]
There is no need to apologize, I'm fine with more complex set up as long as I can find a solution. I did not want to create a new issue as this is some configuration thing on my end for sure.

though adding this to init.lua did not help :(

```
package.path = package.path .. ";" .. vim.fn.expand("$HOME") .. "/.luarocks/share/lua/5.1/?/init.lua;"
package.path = package.path .. ";" .. vim.fn.expand("$HOME") .. "/.luarocks/share/lua/5.1/?.lua;"

```

also some people claim in the discussion that installing pkg-config helped them solve the issue, but it is already on my system
```
$ whereis pkg-config
pkg-config: /usr/bin/pkg-config /usr/share/man/man1/pkg-config.1.gz
```

though I noticed that while the error is the same, the path has changed slightly since adding the package path configurations to init.lua.

```
image.nvim: magick rock not found, please install it and restart your editor. Error: "/home/eagle/.luarocks/share/lua/5.1/magick/wand/lib.lua:220: Failed to load ImageMagick (MagickWand)"
```

 `~/.luarocks/share/lua/5.1/magick` this paths exists and seems to have the rock installed.
